common/controllers/form-wizard.test.js

```diff
@@ -42,7 +42,7 @@ describe('Form wizard', function() {
         const reqMock = {
           t: sinon.stub().returnsArg(0),
         }
-        errors = controller.getErrors(reqMock, {})
+        errors = controller.getErrors(reqMock, {}) // ?
```

Comment:
Was there a reason for this?

Reply:
https://wallabyjs.com/docs/intro/advanced-logging.html#live-comments

Think I'll just stick to console.logging. At least the linter will spot it